[2023-03-28 12:07:11,927] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: defence_dag2.trigger_offence manual__2023-03-28T12:06:58.957363+00:00 [queued]>
[2023-03-28 12:07:11,940] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: defence_dag2.trigger_offence manual__2023-03-28T12:06:58.957363+00:00 [queued]>
[2023-03-28 12:07:11,941] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-03-28 12:07:11,942] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-03-28 12:07:11,942] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-03-28 12:07:11,955] {taskinstance.py:1377} INFO - Executing <Task(TriggerDagRunOperator): trigger_offence> on 2023-03-28 12:06:58.957363+00:00
[2023-03-28 12:07:11,962] {standard_task_runner.py:52} INFO - Started process 1945 to run task
[2023-03-28 12:07:11,965] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'defence_dag2', 'trigger_offence', 'manual__2023-03-28T12:06:58.957363+00:00', '--job-id', '4549', '--raw', '--subdir', 'DAGS_FOLDER/defese.py', '--cfg-path', '/tmp/tmpg1wqpmxk', '--error-file', '/tmp/tmpwyd3jbzx']
[2023-03-28 12:07:11,966] {standard_task_runner.py:80} INFO - Job 4549: Subtask trigger_offence
[2023-03-28 12:07:12,032] {task_command.py:369} INFO - Running <TaskInstance: defence_dag2.trigger_offence manual__2023-03-28T12:06:58.957363+00:00 [running]> on host ff3d40af78f9
[2023-03-28 12:07:12,128] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=defence_dag2
AIRFLOW_CTX_TASK_ID=trigger_offence
AIRFLOW_CTX_EXECUTION_DATE=2023-03-28T12:06:58.957363+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-28T12:06:58.957363+00:00
[2023-03-28 12:07:13,277] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=defence_dag2, task_id=trigger_offence, execution_date=20230328T120658, start_date=20230328T120711, end_date=20230328T120713
[2023-03-28 12:07:13,345] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-03-28 12:07:13,402] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
